lib/eventbrite.rb

```diff
@@ -0,0 +1,50 @@
+require "httparty"
+require_relative "eventbrite/event"
+require_relative "eventbrite/null_event"
+
+class Eventbrite
+  include HTTParty
+
+  base_uri "https://www.eventbriteapi.com/v3"
+
+  def upcoming_event
+    event_list.first
+  end
+
+  private
+
+  def event_list
+    if user_events_response.success? && !user_events_response["events"].empty?
+      user_events_response["events"].map do |event|
+        Eventbrite::Event.new(event)
+      end
+    else
+      [Eventbrite::NullEvent.new]
+    end
+  end
+
+  def user_events_response
+    self.class.get(
+      "/users/#{user_id}/owned_events/",
```

Comment:
What do you think about moving this url into either a method or a local variable just for a descriptive name?